src/scorecard.js

```diff
@@ -0,0 +1,27 @@
+// the following code is for one frame in a scorecard...
+Scorecard = function() {
+  this.bowlOne = [];
+  this.bowlTwo = [];
+  this.frameTotal = [];
+};
+
+Scorecard.prototype.firstBowl = function(num) {
+  this.bowlOne.push(num);
+}
+
+Scorecard.prototype.secondBowl = function(num) {
+  this.bowlTwo.push(num);
+}
+
+Scorecard.prototype.roundSummary = function() {
+  this.frameTotal.push(this.bowlOne[0] + this.bowlTwo[0]);
+}
```

Comment:
It's always better design to have methods that return values, rather than just change the class state.

Reply:
This way, this method and the next can be all rolled into one.